packages/shared/lib/services/sync/job.service.ts

```diff
@@ -148,38 +148,35 @@ export const updateSyncJobResult = async (id: number, result: SyncResultByModel,
 };
 
 export const isSyncJobRunning = async (sync_id: string): Promise<Pick<SyncJob, 'id' | 'job_id' | 'run_id' | 'log_id'> | null> => {
-    const result = await schema()
+    const result = await db.knex
         .from<SyncJob>(SYNC_JOB_TABLE)
+        .select('*')
         .where({
             sync_id,
-            deleted: false,
             status: SyncStatus.RUNNING
         })
         .orderBy('created_at', 'desc')
-        .limit(1);
+        .limit(1)
+        .first();
```

Comment:
aren't `limit(1)` and `first()` the same thing?

Reply:
not exactly, limit is limiting the query, first is picking the first item in the array

Reply:
this is not `arrray.first()` but `queryBuilder.first()`

Reply:
yes it's still pick the first item in the array, but you are right it's also limiting
![Screenshot 2025-03-12 at 09 43 57](https://github.com/user-attachments/assets/9c0abf60-664a-4372-8434-9fa22ca1f7a2)
![Screenshot 2025-03-12 at 09 44 10](https://github.com/user-attachments/assets/ea8191ab-7f74-4488-9c11-f018c74d6923)